frog/imports/ui/GraphEditor/EditorContainer.js

```diff
@@ -74,16 +100,56 @@ class Editor extends Component {
         />
       );
     }
+    const setDelete = val => this.setState({ deleteOpen: val });
+    const setIdRemove = val => this.setState({ idRemove: val });
+    const setImportList = (val, fun?) =>
```

Comment:
Why do you want to manage the importList from here? Is it so that you don't have to refetch every time the user opens the library panel? I guess that makes sense. Make sure there is a reload in that case though - and maybe automatically refetch if more than x minutes have passed.